lib/resources/Authentication.js

```diff
@@ -60,4 +60,15 @@ Authentication.prototype.loginWithKey = function(accessKey, secret, tenantName,
 	});
 };
 
+Authentication.prototype.refreshToken = function(token, scope) {
+	return this.client.request({
+		url : '/rest/platform/authentication/actions/refreshToken',
+		method : 'POST',
+		data: {
+			'refreshToken': token,
+			'scope': scope
```

Comment:
Remover scope, pois deixará de fazer parte das APIs.
```suggestion
			'scope': scope
```